pkg/report/model.go

```diff
@@ -31,6 +31,10 @@ const (
 type LifecycleEvent struct {
 	Type         Event
 	PolicyReport v1alpha2.ReportInterface
+
+	NewItems      map[string]v1alpha2.PolicyReportResult
```

Comment:
should this be already added? its not set yet and as we discussed `ModifiedItems` are not really available

Reply:
Mistake, fixed